src/librustc/hir/mod.rs

```diff
@@ -1008,7 +1008,7 @@ pub enum Mutability {
 }
 
 impl Mutability {
-    /// Returns `MutMutable` only if both arguments are mutable.
+    /// Returns `MutMutable` only if both parameters are mutable.
```

Comment:
Not sure this should be changed, but "both \`self\` and \`other\`" might make more sense.